core-initrd/latest/bin/ubuntu-core-initramfs

```diff
@@ -781,7 +784,10 @@ def create_initrd(parser, args):
                 sys.exit(1)
 
         # Create manifest with packages with files included in the initramfs
-        create_initrd_pkg_list(main, rootfs)
+        manifest_path = create_initrd_pkg_list(main, rootfs)
+        out_dir = os.path.dirname(args.output)
+        manifest_out_path = os.path.join(out_dir, "manifest-initramfs.yaml")
```

Comment:
Could we have an argument for that?

Reply:
The way things are atm are somewhat annoying. I think that we need is actually an "ouput" argument to place all outputs in a single place, I've added a comment about this to not forget.